packages/ffe-css-collapse-react/less/css-collapse.less

```diff
@@ -0,0 +1,3 @@
+.ffe-css-collapse-transition {
+    transition: height @ffe-transition-duration @ffe-ease-in-out-back;
```

Comment:
Maybe we could just inline this style and avoid having a `.less` file at all?
If not then the readme should probably mention that it exists and needs to be imported unless you're importing the `ffe-all.less` file (which was new to me).

Reply:
I thought about it, but what about the less variables? I considered it nice that this component would follow if we change our default duration or easing.

I would rather go for the update of the readme.

Reply:
Fair enough, I forgot about the variable :)

Reply:
I updated the readme and the styleguide-content document.